app/styles/ui/_branches.scss

```diff
@@ -198,6 +199,7 @@
 
 .no-pull-requests {
   width: 365px;
+  margin: var(--spacing) auto;
```

Comment:
Center the empty state content of the Pull Requests tab in the dropdown container, and give its imagery some spacing vertically. 

Read more about the why center empty states below in the Branches tab empty state comments.